uui/components/datePickers/RangeDatePicker.tsx

```diff
@@ -15,9 +15,16 @@ import { i18n } from '../../i18n';
 import css from './RangeDatePicker.module.scss';
 
 export interface RangeDatePickerProps extends BaseRangeDatePickerProps, SizeMod {
+    /**
+     * This callback returns a placeholder depending on the parameters passed.
+     * @param  { from | to }  type
```

Comment:
Please rework it based on our guidelines - https://github.com/epam/UUI/wiki/Documentation-Guidelines#callbacks
This is the case of Pure function callback

Reply:
fixed

Reply:
still doesn't match the guidelines